Source/Core/DolphinQt/Debugger/NetworkWidget.cpp

```diff
@@ -90,6 +90,16 @@ QTableWidgetItem* GetSocketState(s32 host_fd)
   return new QTableWidgetItem(QTableWidget::tr("Unbound"));
 }
 
+static QString GetAddressAndPort(const sockaddr_in& addr)
+{
+  char buffer[16];
+  const char* addr_str = inet_ntop(AF_INET, &addr.sin_addr, buffer, sizeof(buffer));
```

Comment:
What's the advantage of this vs the `inet_ntoa` that you were using before?  Any concerns with the buffer not being large enough?

Reply:
Visual Studio gives me a warning when I use `inet_ntoa`, and that warning gets turned into an error by our build settings. I don't know much beyond that, though I did find this comment when searching for more info about it: https://github.com/SFML/SFML/issues/1023#issuecomment-379424721